src/routing/tests/mod.rs

```diff
@@ -1350,7 +1350,9 @@ async fn handle_untrusted_sync() -> Result<()> {
     }
 
     assert!(bounce_sent);
-    assert_matches!(event_rx.try_recv(), Err(_));
+    assert!(timeout(Duration::from_secs(5), event_rx.recv())
```

Comment:
I understand there is no good way to do this now, but at least make the timeout much shorter so this test doesn't take 5 seconds more than it used to for no good reason. I think 100ms should be more than enough.